src/backend/columnar/sql/columnar--10.0-3--10.1-1.sql

```diff
@@ -20,3 +20,10 @@ END$proc$;
 -- since we dropped pg11 support, we don't need to worry about missing
 -- columnar objects when upgrading postgres
 DROP FUNCTION citus_internal.columnar_ensure_objects_exist();
+
+#include "udfs/columnar_metapage/10.1-1.sql"
+#include "udfs/upgrade_columnar_metapage/10.1-1.sql"
+
+-- upgrade metapage for all columnar relations
+SELECT columnar.upgrade_columnar_metapage(c.oid) FROM pg_class c, pg_am a
+  WHERE c.relam = a.oid AND amname = 'columnar';
```

Comment:
Similarly, should we bump the metapage version back when downgrading citus from 10.1-1 to an older version ?
I don't see any problem with leaving metapage version as "2.0" in older citus versions but still wanted to ask

Reply:
I asked a few questions in chat about this. Hopefully we can resolve it there.

Reply:
I refactored the "upgrade" to be more of an "update" that sets to the current version.